llvm-15.0.3/llvm-crash-analyzer/llvm-crash-analyzer.cpp

```diff
@@ -215,6 +215,12 @@ int main(int argc, char **argv) {
   crash_analyzer::TaintAnalysis TA(TaintDotFileName, MirDotFileName,
                                    PrintPotentialCrashCauseLocation);
   Dec->setTarget(&coreFile.getTarget());
+
+  // Set SBModule for Decompiler, so we can access the symbol table.
+  auto FileSpec = lldb::SBFileSpec(InputFilename.c_str());
+  auto LLDBModule = coreFile.getTarget().FindModule(FileSpec);
```

Comment:
LLDBModule is pointing to main executable. Will this scheme work if the global variable is external to the main executable? For example, a global variable defined in a shared library.
The performance aspect of searching through all symbol table entries may need to watched for big executables.

Reply:
I will investigate further any cases when external global variable is used.
Actually, we can access all linked `LLDBModule`s via `target`, loaded from a core-file,
but I will fix this with in future work.
Speaking about performance, that is also my concern, and I will try to check some bigger cases.
Thanks for your comments!